### DISCUSSION TITLE
Ham Radio, Mac, Need two devices, one each for transmit and receive

### DISCUSSION THREAD

**Discussion by ptbtx** [jackaudio/jack2#778] [Q&A]
## Summary

This is for a Ham Radio environment on a Mac.  I need one device each for TX and RX (transmit and receive).  It's not clear from what I've read so far that I can have two Jack audio devices each connecting two streams (TX and RX) between two ham radio applications.  The receive side is a remote Flex 6xxx radio where the audio is provided locally by xDAX which needs to be routed to the WSJT-X application.  The transmit side is WSJT-X where its audio output needs to be routed to xDAX and then to the remote Flex 6xxx.  Using Loopback from Rogue Amoeba two Virtual Audio Drivers can be created and this works fine (but the cost is $99). Using VB-Cable A+B for macOS from VB-Audio should also work but for some reason the audio is not being routed from WSJT-X to the radio.  Does Jack provide this feature?  Is there a better venue to ask this question?

### Basic example

If the proposal involves a new feature or change in behavior, include a basic code example. Omit this section if it's not applicable.

### Motivation

Why are we doing this? What use cases does it support? What is the expected outcome?

**Comment by falkTX** [comment]
I am moving this to "github discussions" since afaik this is not a bug but a question.
(and on my side I cant answer, dont understand what you want/need anyway. there is a lot of stuff you reference that I have no idea what it is)

**Reply by ptbtx** [reply]
Thank you falkTX.  I provided much more info than needed as I suspect there are some ham radio operators in this group.  To simplify:  Can there be two independent jack devices running at the same time?  The connection for each device would be application to application.
